apps/editors/forms.py

```diff
@@ -331,8 +330,7 @@ def clean_addon_files(self):
 
 def get_review_form(data, request=None, addon=None, version=None):
     helper = ReviewHelper(request=request, addon=addon, version=version)
-    form = {ReviewAddon: ReviewAddonForm,
-            ReviewFiles: ReviewFileForm}[helper.handler.__class__]
+    form = {ReviewAddon: ReviewAddonForm}[helper.handler.__class__]
```

Comment:
Dict is no longer necessary. Just get rid of `form` and use `ReviewAddonForm` directly on the line below.